compiler/tests-common/tests/org/jetbrains/kotlin/jvm/compiler/ir/javac/AbstractIrLoadJavaUsingJavacTest.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2010-2019 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.jvm.compiler.ir.javac
+
+import org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment
+import org.jetbrains.kotlin.config.CompilerConfiguration
+import org.jetbrains.kotlin.config.JVMConfigurationKeys
+import org.jetbrains.kotlin.jvm.compiler.ir.AbstractIrLoadJavaTest
+import org.jetbrains.kotlin.test.KotlinTestUtils
+import java.io.File
+
+abstract class AbstractIrLoadJavaUsingJavacTest : AbstractIrLoadJavaTest() {
```

Comment:
Why not extend AbstractLoadJavaUsingJavacTest?

Reply:
Good point. Done!